cfgov/unprocessed/apps/youth-employment-success/js/money.js

```diff
@@ -0,0 +1,41 @@
+const _centsPerDollar = 100;
+const _decimals = 2;
+const _dollarsToPrecisionRegexp = new RegExp( `(\\d+\\.?\\d{0,${ _decimals }})` );
+
+/**
+ * Converts an input string into a scaled dollar value, or zero.
+ * @param {String} dollars Amount as string
```

Comment:
Trivial, should be `{string}` and `{number}` in the jsdoc types since you're taking and returning a string and number primitive, respectively, not an instance of String and Number.

Reply:
I actually did not realize that, thanks!